test_suite/cloud/test_aws.py

```diff
@@ -355,12 +355,20 @@ def test_hostkey_permissions(self, host):
         """
         Check that ssh files permission set are correct.
         BugZilla 2013644
+
+        Excluded on Fedora >= 38 because the
         """
         files_to_check = ['ssh_host_ecdsa_key',
                           'ssh_host_ed25519_key', 'ssh_host_rsa_key']
+        expected_mode = 0o640
+        if host.system_info.distribution == 'fedora' and int(host.system_info.release) >= 38:
```

Comment:
Will this change get into RHEL? If so, can we already know which version will include this change?

Reply:
My wild guess: Red Hat didn't rebase openssh during the whole lifecycle of RHEL 8, thus I don't expect this to happen neither in RHEL 9 unless someone explicitly requests it. I don't think that's quite unlikely, though. So my guess is it will appear in RHEL 10.